website/src/pages/influenza-b/index.astro

```diff
@@ -1,6 +1,13 @@
 ---
-import GenericOverview from '../../components/views/overview/GenericOverview.astro';
+import LinkedOverview from '../../components/views/overview/LinkedOverview.astro';
+import { getPathogenMegaMenuSections } from '../../layouts/base/header/getPathogenMegaMenuSections';
 import { Organisms } from '../../types/Organism';
+
+const lineages: (keyof ReturnType<typeof getPathogenMegaMenuSections>)[] = ['victoria'];
```

Comment:
Let's please use the constants from `Organisms`. (Also I think you won't need explicit type annotations then, because TS will recognize them as const values. Nitpick: IMO we could also inline `lineages`).